### DISCUSSION TITLE
Problems with Web Builder v0.9g for Devtronic_CNC_Controller_V2_ (STM32F411) board

### DISCUSSION THREAD

**Discussion by LEOOZERAN** [grblHAL/core#612] [General]
In the Web Builder v0.9g firmware for the Devtronic_CNC_Controller_V2_ board (STM32F411), the firmware is not generated. When turning on Spindle sync and Log for configuration. Without them, the firmware is generated. FIX THIS PROBLEM! PLEASE!

**Comment by terjeio** [comment]
Should be fixed now.

**Reply by LEOOZERAN** [reply]
Thank you
